idaes/gas_solid_contactors/unit_models/fixed_bed_0D.py

```diff
@@ -0,0 +1,361 @@
+**EnergyBalanceType.enthalpyPhase** - enthalpy balances for each phase,
+**EnergyBalanceType.energyTotal** - single energy balance for material,
+**EnergyBalanceType.energyPhase** - energy balances for each phase.}"""))
+    CONFIG.declare("gas_property_package", ConfigValue(
+        default=None,
+        domain=is_physical_parameter_block,
+        description="Property package to use for gas phase",
+        doc="""Property parameter object used to define property calculations
+for the gas phase, **default** - None.
+**Valid values:** {
+**useDefault** - use default package from parent model or flowsheet,
+**PhysicalParameterObject** - a PhysicalParameterBlock object.}"""))
+    CONFIG.declare("gas_property_package_args", ConfigValue(
+        default=None,
+        domain=is_physical_parameter_block,
+        description="Arguments to use for constructing gas phase "
+        "property packages",
+        doc="""A ConfigBlock with arguments to be passed to a gas phase
+property block(s) and used when constructing these,
+**default** - None.
+**Valid values:** {
+see property package for documentation.}"""))
+    CONFIG.declare("solid_property_package", ConfigValue(
+        default=None,
+        domain=is_physical_parameter_block,
+        description="Property package to use for solid phase",
+        doc="""Property parameter object used to define property calculations
+for the solid phase, **default** - None.
+**Valid values:** {
+**useDefault** - use default package from parent model or flowsheet,
+**PhysicalParameterObject** - a PhysicalParameterBlock object.}"""))
+    CONFIG.declare("solid_property_package_args", ConfigValue(
+        default=None,
+        domain=is_physical_parameter_block,
+        description="Arguments to use for constructing solid phase "
+        "property packages",
+        doc="""A ConfigBlock with arguments to be passed to a solid phase
+property block(s) and used when constructing these,
+**default** - None.
+**Valid values:** {
+see property package for documentation.}"""))
+    CONFIG.declare("reaction_package", ConfigValue(
+        default=None,
+        # Removed domain argument due to limitations in ReactionBase for
+        # heterogeneous systems
+        description="Reaction package to use for unit",
+        doc="""Reaction parameter object used to define reaction calculations,
+**default** - None.
+**Valid values:** {
+**None** - no reaction package,
+**ReactionParameterBlock** - a ReactionParameterBlock object.}"""))
+    CONFIG.declare("reaction_package_args", ConfigBlock(
+        implicit=True,
+        description="Arguments to use for constructing reaction packages",
+        doc="""A ConfigBlock with arguments to be passed to a reaction block(s)
+and used when constructing these,
+**default** - None.
+**Valid values:** {
+see reaction package for documentation.}"""))
+
+    def build(self):
+        # Call UnitModel.build to setup dynamics
+        super(FixedBed0DData, self).build()
```

Comment:
This can be `super().build()` - we no longer support older Python versions which require the longer form.